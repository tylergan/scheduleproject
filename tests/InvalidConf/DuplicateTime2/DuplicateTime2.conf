on Monday at 1100,1200 run /bin/echo Hello
on Monday at 1000,1100 run /bin/echo there