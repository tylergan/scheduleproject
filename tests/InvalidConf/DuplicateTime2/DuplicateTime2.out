error in configuration: on Monday at 1000,1100 run /bin/echo there
